packages/communication-react/src/api-summary.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * `@azure/communication-react` is an npm package that exports the functionality of the Azure Communication Services - UI Library.
```

Comment:
Let's change this comment a bit - talking about this file is to have a single entry point for api review, not used in production